services/datamanager/builtin/builtin.go

```diff
@@ -217,21 +232,10 @@ func getDurationFromHz(captureFrequencyHz float32) time.Duration {
 // Initialize a collector for the component/method or update it if it has previously been created.
 // Return the component/method metadata which is used as a key in the collectors map.
 func (svc *builtIn) initializeOrUpdateCollector(
+	md componentMethodMetadata,
 	attributes dataCaptureConfig) (
```

Comment:
[nit]: Maybe this should just be config then, like above?

Reply:
Done